### DISCUSSION TITLE
RTL feature for Navigation progress

### DISCUSSION THREAD

**Discussion by majeeddl** [mantinedev/mantine#3389] [Feature requests]
I need the possibility of setting rtl for navigation progress component

**Comment by rtivital** [comment]
What exactly does rtl mean in this context?

**Reply by majeeddl** [reply]
For RTL website, when we are going  to use this component , the direction of beginning loading should be start from right to left.

**Reply by rtivital** [reply]
It works exactly like you've described in mantine.dev demos when direction is changed to RTL

**Reply by majeeddl** [reply]
Yes you are right  , I had a misunderstanding.
